js/components/root.js

```diff
@@ -95,15 +95,15 @@
 				}
 
 				if (this.alreadyOnLatest) {
-					return t('caniupdate', 'This app will show results again, once Nextcloud 14 is close.');
+					return t('caniupdate', 'This app will show results again, once Nextcloud 14 is closed.');
 				}
 
 				if (this.appstoreDisabled) {
 					return t('caniupdate', 'Please make sure your config.php does not set <samp>appstoreenabled</samp> to false.');
 				}
 
 				if (this.appstoreFailed) {
-					return t('caniupdate', 'Could not connect to the appstore or the appstore returned no updates at all. Search manually for updates or make sure your server has access to the internet and can connect to the appstore.');
+					return t('caniupdate', 'Could not connect to the store or it returned no upgrades at all. Search manually for updates or make sure your server has access to the Internet and can connect to the store.');
```

Comment:
Wrong words used again

Reply:
In the very least it should be "app store" like elsewhere. I thought "appery" to be a good word. "app store" has a fairly ugly connotation in that it likens it to the very worst of software distribution.